### PR TITLE
build: deployers

### PR DESCRIPTION
**Pull request by Schlagonia**
TODO:

- Add Project Deployer to address provider
- Add Tokenized Strategy to Release Registry?
- Don't require safe factory to be existent to have system work (Release Registry)

### CODE REVIEW

File: contracts/Managers/V3Deployer.sol [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+pragma solidity >=0.8.18;
+
+import {Positions} from "./Positions.sol";
+import {Registry, RegistryFactory} from "../registry/RegistryFactory.sol";
+import {Roles} from "@yearn-vaults/interfaces/Roles.sol";
+import {IVault} from "@yearn-vaults/interfaces/IVault.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IVaultFactory} from "@yearn-vaults/interfaces/IVaultFactory.sol";
+import {ReleaseRegistry} from "../registry/ReleaseRegistry.sol";
```
Comment (wavey0x) [inline contracts/Managers/V3Deployer.sol:11]: nit: consider importing interface only, not the full contract.

**Comment by wavey0x** [comment]
Should deployer include `SplitterFactory.newSplitter()`?

**Comment by Schlagonia** [comment]
> Should deployer include `SplitterFactory.newSplitter()`?

I dont think a splitter is needed by default. And likely unnecessary for most use cases.

**Comment by Schlagonia** [comment]
https://github.com/yearn/vault-periphery/pull/57

**Pull request closed without merging**